package.json

```diff
@@ -55,7 +55,6 @@
     "**/sha3": "^2.0.7",
     "**/solc": "^0.6.4",
     "**/source-map-support": "0.5.16",
-    "**/yargs-parser": "^15.0.1",
```

Comment:
No longer needed as all sub dependencies now resolve to versions which include the vulnerability patch